spec/components/collections/participants_component_spec.rb

```diff
@@ -0,0 +1,19 @@
+# typed: false
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Collections::ParticipantsComponent, type: :component do
+  let(:rendered) { render_inline(described_class.new(collection: collection)) }
+
+  context 'when displaying a collection' do
+    let(:depositors) { collection.depositors.pluck(:email).join(', ') }
+    let(:managers) { collection.managers }
+    let(:collection) { build_stubbed(:collection, :with_depositors) }
+
+    it 'renders the participant component' do
+      expect(rendered.css('table').to_html).to include(managers)
```

Comment:
I'm not sure this is up to date. Collection#managers returns an association. So it's odd we'd see that in the html.  Perhaps do like you did with depositors.

Reply:
You're right, that changed before me. Fixing. Thanks.